### PR TITLE
Desktop View

### PR DESCRIPTION
**Pull request by Suigeneriz**
I created the following:
- A navbar section of the desktop view with media queries
- A project section
- An about me section with a footer

### CODE REVIEW

**Review by ishimwezachee — COMMENTED** [review]
Hi @Suigeneriz , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has linters that are not passing 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: styles.css [hunk 1]
```diff
@@ -297,3 +307,310 @@ footer {
...
+    justify-content: space-between;
+    background-color: #1a2236;
+    color: #fff;
+  }
+
+  .project-info {
+    padding: 50px;
+    height: 120px;
+    margin: 0 auto;
+  }
```
Comment (1GORDON) [inline styles.css:610]: Great job @Suigeneriz 🤝

- [ ] You may please fix this section and try to make your design a replica of the Figma template. You may as well make your design responsive as it reduces to a smaller screen. Most of the elements seem to be overlapping others as the screen size reduces
- Your design
prject yours

- Figma design
project here Figma

File: index.html [hunk 2]
```diff
@@ -38,175 +46,199 @@ <h1>tech<br />Empress</h1>
...
-            <div class="language-info">
-              <span class="language">HTML/CSS</span>
-              <span class="language">Ruby on Rails</span>
-              <span class="language">JavaScript</span>
+          <section class="projects p2">
+            <img
+              src="./images/project2.svg"
+              alt="Project-2"
+              class="project-img"
+            />
```
Comment (1GORDON) [inline index.html:93]: Great UI @Suigeneriz 🤝

- [ ] You may please fix the white space below this image. This makes your design different from the Figma template that was provided in the project requirements.
white space

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @Suigeneriz ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Professional Readme ✔️
- Professional Pull Request ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️

⛔️ You may kindly fix your linters otherwise the next reviewer might not be able to approve your pull request

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (danmainah) [reply index.html:89]: @Suigeneriz great work implementing the changes, kindly fix the white space below the laptop image as previously requested, the gap is still there.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @Suigeneriz , 

Great work on making the changes requested by a previous reviewer 👏  👏 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design


 # Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Suigeneriz** [comment]
@danmainah Please I do not understand the "whitespace issue" you are referring to as my project is not overflowing.

**Comment by danmainah** [comment]
> @danmainah Please I do not understand the "whitespace issue" you are referring to as my project is not overflowing.

Kindly check the image below, it outlines the section requested by the previous reviewer which still exist
image

**Comment by Suigeneriz** [comment]
Thank you, 
It has been fixed.

File: index.html (hunk 2 above)
Comment (danmainah) [reply index.html:89]: @Suigeneriz great work implementing the changes requested.

**Review by danmainah — APPROVED** [review]
Hi @Suigeneriz , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**